linode/manager
# change: [M3-8263] - Rename Distribution to OS

## Description 📝
Rename Distribution to OS to reduce potential confusion between Distribution and Distributed.

Changes will be released separately from Gecko LA/GA since it's just a text change and this simplifies the need for feature flagging and clean up after.

## Changes  🔄
List any change relevant to the reviewer.
- Rename Distribution tab in Linode Create to OS
- Rename Image label from Images to Linux Distribution in the OS tab

## Target release date 🗓️
7/22

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/3245e67a-e262-4021-bd38-c40b2144e607) | ![image](https://github.com/user-attachments/assets/5f83b34b-27a0-4a91-a743-52c02a59d632) |

## How to test 🧪

### Verification steps
(How to verify changes)
- Go to the Linode Create flow and verify the tab name is `OS` and the Image label is `Linux Distribution` for the OS tab. The Image label for other tabs should remain as `Images`

## As an Author I have considered 🤔

*Check all that apply*
